src/NzbDrone.Api/Config/DownloadClientConfigResource.cs

```diff
@@ -15,4 +16,25 @@ public class DownloadClientConfigResource : RestResource
         public bool AutoRedownloadFailed { get; set; }
         public bool RemoveFailedDownloads { get; set; }
     }
+
+    public static class DownloadClientConfigResourceMapper
+    {
+        public static DownloadClientConfigResource MapToResource(IConfigService model)
+        {
+            return new DownloadClientConfigResource
+            {
+                Id = 1,
```

Comment:
I think this might be a big assumption, I don't think it belongs here, if default it always `1` then maybe move it to `DownloadClientConfigResource` constructor, if not I would keep it in the RestModule.